spark/src/main/kotlin/com/adevinta/spark/components/bottomsheet/modal/ModalBottomSheet.kt

```diff
@@ -432,15 +473,22 @@ private fun ModalBottomSheetSample() {
     // Sheet content
     if (openBottomSheet) {
         ModalBottomSheet(
+            showHandle = true,
             onDismissRequest = { openBottomSheet = false },
             sheetState = bottomSheetState,
         ) {
             LazyColumn {
                 stickyHeader {
-                    Row(Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.Center) {
+                    Row(
+                        Modifier
+                            .fillMaxWidth()
+                            .background(Color.Green),
```

Comment:
Is this wanted?

Reply:
Just to highlight the fact that content can fill entire layout,
I will change it to `SparkTheme.colors.background` so it looks more natural